content/en/docs/concepts/architecture/nodes.md

```diff
@@ -80,53 +80,53 @@ For self-registration, the kubelet is started with the following options:
 
     No-op if `register-node` is false.
   - `--node-ip` - IP address of the node.
-  - `--node-labels` - {{< glossary_tooltip text="Labels" term_id="label" >}} to add when registering the node in the cluster (see label restrictions enforced by the [NodeRestriction admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)).
+  - `--node-labels` - {{< glossary_tooltip text="Labels" term_id="label" >}} to add when registering the node in the cluster (see label restrictions enforced by the [`NodeRestriction` admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)).
   - `--node-status-update-frequency` - Specifies how often kubelet posts node status to master.
 
-When the [Node authorization mode](/docs/reference/access-authn-authz/node/) and
-[NodeRestriction admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction) are enabled,
-kubelets are only authorized to create/modify their own Node resource.
+When the [node authorization mode](/docs/reference/access-authn-authz/node/) and
```

Comment:
- I would capitalize Node (line 86)
- Line 88, this is an odd sentence, "... kubelets are only authorized ..."

Reply:
```suggestion
When the [`Node` authorization mode](/docs/reference/access-authn-authz/node/) and
```
or
```markdown
Once you enable the [Node](/docs/reference/access-authn-authz/node/) authorization mode and
```
_plus further changes to match_.

It's “Node” when you configure it in the API server: `--authorization-mode=Node`